WebKitBrowser/CMakeLists.txt

```diff
@@ -118,13 +113,18 @@ target_include_directories(${MODULE_NAME}
         ${WPE_WEBKIT_INCLUDE_DIRS}
         ${WPE_WEBKIT_INCLUDE_DIRS}/WPE
         ${GLIB_INCLUDE_DIRS}
-        ${CMAKE_CURRENT_SOURCE_DIR}
-        ${CMAKE_CURRENT_SOURCE_DIR}/interfaces)
+        #${CMAKE_CURRENT_SOURCE_DIR}
```

Comment:
We need to generate and build a custom interface. Keep this.

Reply:
Can we have a discussion on custom interfaces? The only reason for having an interface is if someone else is using it, would this be an interface between the WebKitPlugin and the injected bundle ?